src/main/java/com/example/report/service/ReportService.java

```diff
@@ -119,11 +133,120 @@ public List<ReportEntryDTO> generateReportEntries(LocalDate from, LocalDate to)
         return reportEntryMapper.entityToDto(savedReportEntries);
     }
 
-    public List<ReportDTO> getAllReports(){
+    public List<ReportDTO> getAllReports() {
         return reportMapper.entityToDto(reportRepository.findAll());
     }
 
-    public List<ReportEntryDTO> getReportEntriesByReportId(Long reportId){
+    public List<ReportEntryDTO> getReportEntriesByReportId(Long reportId) {
         return reportEntryMapper.entityToDto(reportEntryRepository.findAllByReportId(reportId));
     }
+
+
+    private void writeReportDataInExcelRow(Sheet sheet, Integer rowNumber, Report report, CellStyle numberStyle, CellStyle dateStyle){
+        Row row = sheet.createRow(rowNumber);
+
+        Cell cell = row.createCell(0);
+        cell.setCellValue(report.getId());
+        cell.setCellStyle(numberStyle);
+
+        cell = row.createCell(1);
+        cell.setCellValue(report.getStartDate());
+        cell.setCellStyle(dateStyle);
+
+        cell = row.createCell(2);
+        cell.setCellValue(report.getEndDate());
+        cell.setCellStyle(dateStyle);
+    }
+
+    private void applyCommonSheetStyle(Sheet sheet){
+        sheet.autoSizeColumn(0);
+        sheet.autoSizeColumn(1);
+        sheet.autoSizeColumn(2);
+        sheet.createFreezePane(0,1);
+        sheet.setDisplayGridlines(false);
+        sheet.setZoom(130);
+    }
+
+    private CellStyle headerCellStyle(Workbook workbook, Sheet sheet){
```

Comment:
If we return headerCellStyle then the method should be called `getHeaderCellStyle()` or something like that